libevm/JitVM.cpp

```diff
@@ -175,9 +174,27 @@ void create(evm_result* o_result, ExtVMFace& _env, evm_message const* _msg) noex
 	}
 	else
 	{
-		o_result->code = EVM_FAILURE;
-		o_result->output_data = nullptr;
-		o_result->output_size = 0;
+		o_result->code = EVM_REVERT;
```

Comment:
I meant that here `ExtVMFace::create` could not only succeed or revert, but also fail for other reason. But you always return `EVM_REVERT` in case of failure, so EVM will try to use return buffer in case of OOG for example. Will that be a problem?